src/App.jsx

```diff
@@ -1,64 +1,34 @@
-import { useState } from 'react'
 import { BrowserRouter, Routes, Route, Link } from 'react-router-dom'
-import { deleteCocktail } from './services/cocktails.service'
+import { useSelector } from 'react-redux'
 
 import Home from './pages/Home'
 import Create from './pages/Create'
 import Update from './pages/Update'
 
 import CocktailModal from './components/CocktailModal'
+import Logo from './cover.png'
 
 function App() {
-  const [cocktails, setCocktails] = useState(null)
-  const [showModal, setShowModal] = useState(false)
-  const [currentModalCocktail, setCurrentModalCocktail] = useState(null)
+  const { showModal } = useSelector((state) => state.modal)
```

Comment:
Since you're using `@reduxjs/toolkit` I believe this could be abstracted further, into its own selector — refer to: https://redux-toolkit.js.org/api/createSelector and check out https://redux-toolkit.js.org/api/createSelector

Basically, you can define a list of selectors, and just use them here, something like:

```
const showModal = useSelector(selectShowModal);
```

Where `selectShowModal` would select the appropriate state

This can be kinda weird to get at first, so feel free to skip it :) But the main point is: this component probably shouldn't be concerned with how the modal state is selected from the root state. What if you'd need to show the same modal somewhere else? That other component would also have to know the logic of selecting the right state

Reply:
Also, maybe the name of the variable can be more specific, how about `showCockailModal`?

Reply:
Alright, I've seen that term while going through some code. I'll see if I can apply it !